Loads/Loads/Point2dMoment.cs

```diff
@@ -0,0 +1,18 @@
+using OasysUnits;
+
+namespace MagmaWorks.Taxonomy.Loads
+{
+    public class Point2dMoment : IPoint2dMoment
+    {
+        public Moment Yy { get; set; } = Moment.Zero;
+        public Moment Zz { get; set; } = Moment.Zero;
+
+        internal Point2dMoment() { }
```

Comment:
We actually could make the default constructor public?

Reply:
I didnt want to have an empty constructor initialising all properties to zero. But I needed a default emptry constructor to inherit from this class, and also the serialization prefers a default empty constructor